addons/udes_stock/models/stock_move.py

```diff
@@ -146,8 +146,13 @@ def split_out_move_lines(self, move_lines):
     def _action_assign(self):
         res = super(StockMove, self)._action_assign()
         for picking_type in self.mapped('picking_type_id'):
-            self.filtered(lambda m: m.picking_type_id == picking_type). \
-                post_reservation_split()
+            moves = self.filtered(lambda m: m.picking_type_id == picking_type)
```

Comment:
Above, perhaps it's better to use `_` instead of defining `res` as it's not used.

Reply:
Nitpick: I would rename `picking_type` to `picking_type_id` to improve readability.

Reply:
Can we add a brief docstring to `_action_assing`?

Reply:
Looking at the `picking::action_assign` method: do we need to validate the `res` returned by the parent class method?